docs/ml/index.asciidoc

```diff
@@ -1,35 +1,36 @@
 [role="xpack"]
 [[xpack-ml]]
-= Machine Learning
+= {ml-cap} {anomaly-detect}
 
 [partintro]
 --
 
 As datasets increase in size and complexity, the human effort required to
 inspect dashboards or maintain rules for spotting infrastructure problems,
-cyber attacks, or business issues becomes impractical. The Elastic {ml-features}
-automatically model the normal behavior of your time series data — learning
-trends, periodicity, and more — in real time to identify anomalies, streamline
-root cause analysis, and reduce false positives.
+cyber attacks, or business issues becomes impractical. The Elastic {ml} 
+{anomaly-detect} feature automatically model the normal behavior of your time 
+series data — learning trends, periodicity, and more — in real time to identify 
+anomalies, streamline root cause analysis, and reduce false positives.
 
-The {ml-features} run in and scale with {es}, and include an
-intuitive UI on the {kib} *Machine Learning* page for creating anomaly detection
-jobs and understanding results.
+{anomaly-detect-cap} runs in and scales with {es}, and include an
+intuitive UI on the {kib} *Machine Learning* page for creating {anomaly-jobs} 
+and understanding results.
 
 If you have a basic license, you can use the *Data Visualizer* to learn more
 about your data. In particular, if your data is stored in {es} and contains a
 time field, you can use the *Data Visualizer* to identify possible fields for
-{ml} analysis:
+{anomaly-detect}:
 
 [role="screenshot"]
 image::ml/images/ml-data-visualizer-sample.jpg[Data Visualizer for sample flight data]
 
-experimental[] You can also upload a CSV, NDJSON, or log file (up to 100 MB in size). 
-The {ml-features} identify the file format and field mappings. You can then 
-optionally import that data into an {es} index.  
+experimental[] You can also upload a CSV, NDJSON, or log file (up to 100 MB in 
+size). {anomaly-detect-cap} identifies the file format and field mappings. You 
```

Comment:
This is on the Data Visualizer tab, so I don't think it's related to anomaly detection:
```suggestion
size). The *Data Visualizer* identifies the file format and field mappings. You 
```